workers/sendWeeklyRecapEmails.js

```diff
@@ -161,12 +161,18 @@ var worker = function(job, done){
 
             variables.WEAK_HABITS = 0;
             variables.STRONG_HABITS = 0;
+            var weakHabitCutoff = -1.1;
```

Comment:
This would probably benefit from a brief comment like `// value determined experimentally by clicking a negative-only Habit twice`  (or 'once' or whatever becomes appropriate if the code is changed). Otherwise we'll wonder in future how we got this number. :)